front/pages/api/login.ts

```diff
@@ -47,7 +47,7 @@ async function handleMembershipInvite(
     AuthFlowError | SSOEnforcedError
   >
 > {
-  if (membershipInvite.inviteEmail !== user.email) {
+  if (membershipInvite.inviteEmail.toLowerCase() !== user.email.toLowerCase()) {
```

Comment:
I'd suggest that we eliminate the need to consistently use `.toLowerCase()` on email addresses, as it is easy to overlook. Instead, we should store all email addresses in lowercase for both users and invitations. FWIW, most email servers treat uppercase and lowercase as equivalent.